### DISCUSSION TITLE
Commands

### DISCUSSION THREAD

**Discussion by DenisPryt** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#114] [Q&A] [answered]
Hi! Thanks for your work! I have a question about commands.
In [attributes](https://raw.githubusercontent.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/master/images/map_card.gif)  section I see ability to send commands to Vacuum, but I don't have drop down on my dashboard. 
And I don't understand what a purpose of `Attributes configuration`? All of these attributes have something to do with [lovelace card](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card)?

**Comment by PiotrMachowski** [comment, marked as answer]
Adding `attributes` to your config adds them to camera entity: if you want your camera to have an attribute containing room numbers you have to add `room_numbers` etc. Adding `calibration_points` (like in [recommended config](https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#recommended)) adds a set of calibration points for [lovelace card](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card), which enables automatic card calibration. To have an ability to send commands to vacuum like on [this gif](https://raw.githubusercontent.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/master/images/map_card.gif) you have to install said card.
